templates/desktop/index.html

```diff
@@ -6,15 +6,29 @@
 {% block content %}
 {% include "desktop/secondary-nav.html" %}
 
+<!--fix the video title cropped-->
+<style>
+  @media (min-width: 1036px){
+    .bilibili_title {
+      width: 90%;
+    }
+  }
+</style>
+
 <section class="p-strip--image is-dark is-deep u-no-padding--top u-no-padding--bottom desktop-hero">
-    <div class="p-strip--suru-background">
+    <div class="p-strip--suru">
       <div class="row">
         <div class="col-6">
           <h1>Ubuntu桌面系统</h1>
           <p class="desktop-hero--subtext">开源的Ubuntu桌面操作系统为全球数百万的PC和笔记本电脑提供了生产力。</p>
           <p><a href="/download/desktop" class="p-button--positive">下载Ubuntu</a></p>
           <p><a href="/desktop/features" class="p-link--inverted">了解功能特点&nbsp;&rsaquo;</a></p>
         </div>
+        <div class="col-6 u-vertically-center bilibili_title">
+          <div class="u-embedded-media">
+            <iframe class="u-embedded-media__element" width="560" height="315" src="//player.bilibili.com/player.html?aid=645535740&bvid=BV1fY4y1K793&cid=833405913&page=1&high_quality=1&danmaku=0" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>

```

Comment:
`title="YouTube video player"` it's not a Youtube video player? Can you change it something else, maybe in Chinese.

Reply:
Yes, it's not a Youtube player because of the access issue. The "title" doesn't display with a non-Youtube player link, so it's fine I think.